src/pages/gen2/build-a-backend/auth/enable-sign-up/index.mdx

```diff
@@ -174,8 +174,8 @@ Using password policy in your `amplify/auth/resource.ts` file, you can customize
 ```ts title="amplify/backend.ts"
 // amplify/backend.ts
 import { defineBackend } from '@aws-amplify/backend';
-import { auth } from './auth/resource.js';
```

Comment:
Do we know why this was done in the first place?

Reply:
I think this was before we were using `moduleResolution: nodenext` instead of `bundler`, which forces the explicit file extension for Node ESM. The backend is using tsx, which allows us to switch to `bundler` and work around the explicit file extensions